MAINTAINERS.yml

```diff
@@ -793,6 +826,9 @@ Little FS:
     description: >-
        Little FS
 
+    labels:
+        - "area: File System"
```

Comment:
Could this be specific to "Little FS"

Reply:
IMO this is going to be too specific, if we start adding labels for each implementation.

Reply:
@ioannisg I agree with @nashif. I have one comment though, there is "vfs" which is also listed under "FIle System" area and it contains driver for FAT file system; if the LittleFS got its own section, then maybe the FAT FS should also get one.
VFS itself should contain file system API common files that are not file system specific, IMO.

Reply:
> there is "vfs" which is also listed under "FIle System" area and it contains driver for FAT file system;

yes, I think we can move FAT into its own entry